examples/vpc-public-without-bastion/main.tf

```diff
@@ -2,15 +2,14 @@
 # Terraform Settings
 # -------------------------------------------------------------------------------------------------
 terraform {
-  # >= v0.11.10 AND < v0.12
-  required_version = ">= 0.11.10, ~> 0.11"
+  required_version = ">= 0.12.30"
 }
 
 # -------------------------------------------------------------------------------------------------
 # AWS Settings
 # -------------------------------------------------------------------------------------------------
 provider "aws" {
-  version = "~> 1.43"
+  version = "~> 3.35"
```

Comment:
Do you happen to know why? Could it be because of the sops - terragrunt integration? I seem to recall something along those lines...
Anyhow, I don't see that being a problem. Whenever the other repos will be adjusted so that they will support newer versions of the AWS provider, they can simply bump the release tag of this VPC module. Until then, they can simply stay on the `v1.0.1` tag.

**Update:** It turns out that this module is not actually used in the `lite-infra` repo. Are there any other repos with this provider pinning that you know of?
Also, I found your commit that mentioned a bug in SSM for that version. It seems that in `v3.36.0`, some SSM stuff was addressed: https://github.com/hashicorp/terraform-provider-aws/releases/tag/v3.36.0
Perhaps you might wanna give that a go.

Reply:
As it is just an example - let's skip it